nuxeo-importer-stream/src/main/java/org/nuxeo/importer/stream/consumer/BlobMessageConsumer.java

```diff
@@ -99,4 +122,37 @@ public void commit() {
     public void rollback() {
 
     }
+
+    public class MyBlob {
+        protected final Blob blob;
+
+        protected final String fileToDelete;
```

Comment:
You can make this a `File` or `Path` object directly to avoid a round-trip through the string representation.